### PR TITLE
feat: contact information in dashboard

### PR DESCRIPTION
**Pull request by djaiss**
This adds the contact information changes in the dashboard feed.

image

### CODE REVIEW

File: domains/Contact/ManageContactFeed/Web/ViewHelpers/Actions/ActionFeedContactInformation.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        $contact = $item->contact;
+        $contactInformation = $item->feedable;
+
+        return [
+            'information' => [
+                'object' => $contactInformation ? [
+                    'id' => $contactInformation->id,
+                    'label' => $contactInformation->name,
+                    'data' => $contactInformation->contactInformationType->protocol ? $contactInformation->contactInformationType->protocol.$contactInformation->data : $contactInformation->data,
```
Comment (asbiin) [inline domains/Contact/ManageContactFeed/Web/ViewHelpers/Actions/ActionFeedContactInformation.php:19]: you should be able to write it, but I did not tested it:
```suggestion
                    'data' => optional($contactInformation->contactInformationType->protocol).$contactInformation->data,
```

File: resources/js/Shared/Modules/FeedItems/ContactInformation.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      <!-- the contact information still exists in the system -->
+      <span v-if="data.information.object">
+        <a :href="data.information.object.data" class="text-blue-500 hover:underline"
+          >{{ data.information.object.label }}
+        </a>
+      </span>
+
+      <!-- the label was deleted -->
+      <span v-else>
+        <span>{{ data.information.description }}</span>
```
Comment (asbiin) [inline resources/js/Shared/Modules/FeedItems/ContactInformation.vue]: do you need this 2nd `span` here?
```suggestion
        {{ data.information.description }}
```

**Review by asbiin — APPROVED** [review]
See my comments

File: domains/Contact/ManageContactFeed/Web/ViewHelpers/Actions/ActionFeedContactInformation.php (hunk 1 above)
Comment (djaiss) [reply domains/Contact/ManageContactFeed/Web/ViewHelpers/Actions/ActionFeedContactInformation.php:19]: Unfortunately that doesn't work. I understand the intention though.

**Pull request merged**